src/plugins/intel_cpu/src/snippets_transformations/op/load_convert.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+#include "snippets/op/load.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+/**
+ * @interface LoadConvertSaturation
+ * @brief Generated for load and convert (with saturation) at the same time
```

Comment:
Fused operation to represent computations equal to consecutive Load and Convert operations.
The operation is used for peephole optimization during subgraph lowering.